lighthouse-core/audits/autocomplete.js

```diff
@@ -43,30 +185,83 @@ class AutocompleteAudit extends Audit {
     };
   }
 
+  /**
+   * @param {LH.Artifacts.FormInput} input
+   * @return {{isValid: Boolean, orderWarn?: Boolean}}
+   */
+  static isValidAutocomplete(input) {
+    if (!input.autocomplete.attribute) return {isValid: false};
+    if (input.autocomplete.attribute.includes(' ') ) {
+      const tokenArray = input.autocomplete.attribute.split(' ');
+      for (const token of tokenArray) {
+        if (token.slice(0, 8) === 'section-') continue;
+        if (validAutocompleteTokens.includes(token)) continue;
+        return {isValid: false};
+      }
+      if (!input.autocomplete.property) return {isValid: false, orderWarn: true};
+      return {isValid: true};
+    }
+    return {isValid: validAutocompleteTokens.includes(input.autocomplete.attribute)};
+  }
+
   /**
    * @param {LH.Artifacts} artifacts
    * @return {LH.Audit.Product}
    */
   static audit(artifacts) {
     const forms = artifacts.FormElements;
     const failingFormsData = [];
+    const warnings = [];
+    let notApplicable = 0;
+    let inputsCount = 0;
     for (const form of forms) {
       for (const input of form.inputs) {
-        if (!input.autocomplete) {
-          failingFormsData.push({
-            node: /** @type {LH.Audit.Details.NodeValue} */ ({
-              type: 'node',
-              snippet: input.snippet,
-              nodeLabel: input.nodeLabel,
-            }),
-          });
+        inputsCount += 1;
+        const token = this.isValidAutocomplete(input);
+        if (!token.isValid) {
+          if (!input.autocomplete.prediction) {
+            notApplicable += 1;
+            continue;
+          }
+          if (noPrediction.includes(input.autocomplete.prediction) &&
+          !input.autocomplete.attribute) {
+            notApplicable += 1;
+            continue;
+          }
+          if (input.autocomplete.prediction in autofillSuggestions) {
```

Comment:
not sure what this conditional is really checking... is it meant to be about the prediction matching?  or did you want to handle the case where the prediction returned isn't in our list of `autofillSuggestions`

i'm happy to help think through the logic if you want.

Reply:
for the case if its not in our list of autocomplete suggestions. but I just thought of a better way to do it

Reply:
okay just updated it. also is there anyway to catch that data for us to get it so we will be able to increase our coverage